### ISSUE TITLE
[Effect Suggestion] Smile, You're On Camera

### ISSUE THREAD

**Issue by TerrizX** [gta-chaos-mod/ChaosModV#2393]
**Duration (choose one and replace the corresponding "[ ]" with "[X]")**
- [X] Long
- [ ] Short
- [ ] No duration

**Effect Description**
Main camera becomes the view of the cameras in the game that face the player like well as electronic billboards, ATMs, car dashboard cameras and CCTV cameras.

**Comment by Last0xygen** [comment]
So like „news Team?

**Comment by TerrizX** [comment]
> So like „news Team?

Nah, the camera view would be only the cameras in the players vicinity. Like an ongoing TV show or something

**Comment by TerrizX** [comment]
> So like „news Team?

> So like „news Team?

Nah, the camera view would be only the cameras in the players vicinity. Like an ongoing TV show or something. Think Watch Dogs when u hack into cameras in that game and see through those. It would be that but if the player is closer to said camera, the game would switch automatically to said camera

**Comment by turg-the-sad-cat** [comment]
But there are no cameras like that In gta 5, if there is there’s only a few places for that and even then you probably won’t be near it to have that camera

**Comment by DaBestIvo** [comment]
Sounds really cool, but every single camera would probably have to be manually added.
